src/include/function/table/bind_data.h

```diff
@@ -17,18 +17,24 @@ struct KUZU_API TableFuncBindData {
     // the last {numWarningDataColumns} columns are for temporary internal use
     common::column_id_t numWarningDataColumns;
     common::cardinality_t cardinality;
+    common::offset_t maxOffset;
```

Comment:
I would make the default to be INVALID_OFFSET

Reply:
hmm I think 0 is more safe here, as `TableFuncSharedState` would use this to dispatch morsels.